examples/7_explorer.py

```diff
@@ -189,9 +189,13 @@ def run(self):
     agent = Explorer(miss, visualizer=visualizer)
     sleep(4)
 
-    # minelogy initialization with current minecraft version
-    mcver = agent.getVersion()
-    mlogy = Minelogy(mcver) # to initialize for usage in this script
+    # initialize minelogy
+    agent.rob.update_in_background()
+    agent.rob.sendCommand('item_list')
+    sleep(2)
+    item_list = agent.rob.mc.observe[0]['item_list']
```

Comment:
i would add mc.getItemList method, like for other observations

Reply:
@noskill  Done. Though, we don't have getRecipe method, for example.